### PR TITLE
javascript-loops

### PR DESCRIPTION
**Pull request by KennethDelaCruz**
image

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            getValues
+ */
+function getNumbersToTen() {
+  const numbers = [];
+  let currentNumber = 1;
+
+  for (let i = 0; i < 10; i++) {
+    numbers.push(currentNumber);
+    currentNumber = currentNumber + 1;
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js]: This should be a `while` loop

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+console.log(getNumbersToTen(0));
+
+function getNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+
+  for (let i = 0; i < 10; i++) {
+    evenNumbers.push(currentNumber);
+    currentNumber = currentNumber + 2;
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js]: This should be a `while` loop

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Kenn! A couple changes here. Also, you have `const` and `let` in your code. Did you do this deliberately, or was it changed by your text editor? You may still need to complete proper configuration of your `.eslintrc` file, or you still need to merge the pull request that includes that change. Please maker sure to merge any approved pull requests

**Comment by KennethDelaCruz** [comment]
Hi Cody, The changes for the Const and Let were not intentional. It was probably the text editor. I went back and merged all the exercises that need to be merged. Hopefully that fixes it.

**Comment by KennethDelaCruz** [comment]
here are the two changes that you have asked for. I have also changed all the let/const be changed to vars again. 

image
image

**Review by codingcodymiller — APPROVED** [review]
🙌

**Pull request merged**